### ISSUE TITLE
Error: The 'elm-make' compiler is not available

### ISSUE THREAD

**Issue by jpierson** [sbrink/vscode-elm#61]
I've installed elm for the first time trying to learn via [this tutorial](http://www.elm-tutorial.org/en/01-foundations/01-hello.html) and when opening the Hello.elm file in the first section I get the following error in VSCode.

> The 'elm-make' compiler is not available. Install Elm from http://elm-lang.org/

Here is my version of VSCode, the elm extension is also up to date. I'm running on Windows 10.

```
Version 1.2.1
Commit fe7f407b95b7f78405846188259504b34ef72761
Date 2016-06-14T13:00:22.146Z
Shell 0.37.6
Renderer 49.0.2623.75
Node 5.10.0
```

Elm was installed via Windows installer `Elm-Platform-0.17.exe` downloaded from http://elm-lang.org/.

**Comment by stevensona** [comment]
Having the same issue using elm installed via npm.

**Comment by Krzysztof-Cieslak** [comment]
Fixed by #63
